debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/transforms/MongoDataConverterTest.java

```diff
@@ -61,29 +61,28 @@ public void shouldCreateCorrectStructFromInsertJson() {
 
         assertThat(struct.toString()).isEqualTo(
                 "Struct{"
-                + "address=Struct{"
-                  + "building=1007,"
-                  + "floor=Struct{"
-                    + "level=17,"
-                    + "description=level 17"
-                  + "},"
-                  + "coord=[-73.856077, 40.848447],"
-                  + "street=Morris Park Ave,"
-                  + "zipcode=10462"
-                + "},"
-                + "borough=Bronx,"
-                + "cuisine=Bakery,"
-                + "grades=["
-                  + "Struct{date=1393804800000,grade=A,score=2}, "
-                  + "Struct{date=1378857600000,grade=A,score=6}, "
-                  + "Struct{date=1358985600000,grade=A,score=10}, "
-                  + "Struct{date=1322006400000,grade=A,score=9}, "
-                  + "Struct{date=1299715200000,grade=B,score=14}"
-                + "],"
-                + "name=Morris Park Bake Shop,"
-                + "restaurant_id=30075445"
-              + "}"
-        );
+                        + "address=Struct{"
```

Comment:
@Naros, this reformatting is a bit unfortunate as it looses all the (JSON) indentation. Granted, the indenting whitespace probably should have been *within* the quotes (as it's the case for some other tests). It's the case and and in some other tests, too. Would you perhaps be up for fixing this?

Reply:
We have a few options for tests like this:

1. We mark specific tests like this as ignored by the formatter 
2. We use formatter comment blocks to basically mark areas of the code to skip formatting

Personally I prefer (2) and it would look something like this when used:
```
// @formatter:off
...
// @formatter:on
```
We all agree this is probably the best path forward for now?

Reply:
@gunnarmorling I pushed a commit that should resolve these from what I saw.  I decided for most of the scenarios that it made sense to use the formatter on/off tags for now.